torch/_dynamo/guards.py

```diff
@@ -637,13 +635,8 @@ def direct_negation(a, b):
                 ("___check_tensors", check_tensors_fn),
                 ("___check_tensors_verbose", check_tensors_verbose_fn),
                 ("tensor_check_names", tensor_check_names),
-                ("floor", math.floor),
-                ("ceiling", math.ceil),
-                ("Eq", direct_equality),
```

Comment:
Delete direct equality and negation? These were just to remap and it looks like we aren't using them anymore?